master/internal/stream/model_versions.go

```diff
@@ -171,14 +176,14 @@ func ModelVersionCollectStartupMsgs(
 			query = modelVersionPermFilterQuery(query, accessScopes)
 		}
 		err := query.Scan(ctx, &mvMsgs)
-		if err != nil && errors.Cause(err) != sql.ErrNoRows {
+		if err != nil && !errors.Is(err, sql.ErrNoRows) {
 			log.Errorf("error: %v\n", err)
 			return nil, err
```

Comment:
nit: is there a reason we aren't wrapping this error?

Reply:
Will wrap it.